File: Modul 3/Soal_1/PRAK301-2410817310003-Muhammad_Guntur_Ricky_Adhitya.py
inp = input()
input_1, input_2, input_3 = inp.split()

if input_1 > input_2 :
    input_1, input_2 = input_2, input_1

if input_2 > input_3 :
    input_2, input_3 = input_3, input_2

if input_1 > input_2 :
    input_1, input_2 = input_2, input_1

print(input_1, input_2, input_3)

File: Modul 3/Soal_1/PRAK301-2410817310003-Muhammad_Guntur_Ricky_Adhitya.c
#include <stdio.h>

int main(){
    int input_1, input_2, input_3;
    scanf("%d %d %d", &input_1, &input_2, &input_3);

    int temp;
    if(input_1 > input_2){
        temp = input_1;
        input_1 = input_2;
        input_2 = temp;
    }

    if(input_2 > input_3){
        temp = input_2;
        input_2 = input_3;
        input_3 = temp;
    }

    if(input_1 > input_2){
        temp = input_1;
        input_1 = input_2;
        input_2 = temp;
    }

    printf("%d %d %d", input_1, input_2, input_3);
    return 0;
}